docs/tutorial/part-five/index.md

```diff
@@ -1,53 +1,53 @@
 ---
-title: Source Plugins
+title: Les plugins source
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-> This tutorial is part of a series about Gatsby’s data layer. Make sure you’ve gone through [part 4](/tutorial/part-four/) before continuing here.
+> Ce tutoriel fait partie d'une série concernant la couche de données de Gatsby. Assurez-vous d'avoir suivi la [partie 4](/tutorial/part-four/) avant de poursuivre ici.
 
-## What's in this tutorial?
+## Que contient ce tutoriel ?
 
-In this tutorial, you'll be learning about how to pull data into your Gatsby site using GraphQL and source plugins. Before you learn about these plugins, however, you'll want to know how to use something called GraphiQL, a tool that helps you structure your queries correctly.
+Dans ce tutoriel, vous apprendrez comment intégrer des données dans votre site Gatsby en utilisant GraphQL et des plugins source. Avant de vous pencher sur ces plugins cependant, vous devez apprendre à vous servir d'une chose qui s'appelle GraphiQL, un outil qui vous aide à structurer vos requêtes correctement.
 
-## Introducing GraphiQL
+## Introduction à GraphiQL
 
-GraphiQL is the GraphQL integrated development environment (IDE). It's a powerful (and all-around awesome) tool you'll use often while building Gatsby websites.
+GraphiQL est l'environnement de développement intégré (EDI) de GraphQL. C'est un outil puissant (et en tout point génial) que vous serez souvent amené à utiliser en construisant des sites web Gatsby.
 
-You can access it when your site's development server is running—normally at
+Vous pouvez y accéder lorsque votre serveur de développement est lancé - normalement ici
 <http://localhost:8000/___graphql>.
 
 <video controls="controls" autoplay="true" loop="true">
   <source type="video/mp4" src="/graphiql-explore.mp4"></source>
-  <p>Your browser does not support the video element.</p>
+  <p>Votre navigateur ne prend pas en charge cet élément vidéo</p>
```

Comment:
I think "**the** video element" is more for the HTML tag. It is weird in French to talk implicitly about this as in English so I added some context.

```suggestion
  <p>Votre navigateur ne prend pas en charge l'élément HTML5 "vidéo"</p>
```

Reply:
We could also say "Please stop using IE6-8" but as it is not mentioned in the original version ... 😜